apps/src/templates/teacherDashboard/LoginTypePicker.jsx

```diff
@@ -52,8 +60,13 @@ class LoginTypePicker extends Component {
     this.props.setLoginType(provider);
   };
 
+  cancel = () => {
```

Comment:
Because the cancel function is passed in as a prop, I just combined these two calls the same way we do in RosterDialog.jsx